cmd/mattermost-mattermod/main.go

```diff
@@ -30,11 +30,11 @@ func main() {
 	config, err := server.GetConfig(configFile)
 	if err != nil {
 		mlog.Error("unable to load server config", mlog.Err(err), mlog.String("file", configFile))
-		os.Exit(1)
```

Comment:
i think this should stay as `os.Exit(1)`
same for the others

Reply:
I followed this suggestion: https://github.com/mattermost/mattermost-mattermod/pull/275#discussion_r727288928